### PR TITLE
Preview of impact from PR gstreamer-rs#78

### PR DESCRIPTION
**Pull request by fengalin**
This is a preview of the impact from the proposal in https://github.com/sdroege/gstreamer-rs/pull/78

### CODE REVIEW

File: gst-plugin-togglerecord/src/togglerecord.rs [hunk 1]
```diff
@@ -1054,7 +1054,7 @@ impl ToggleRecord {
 
         gst_log!(self.cat, obj: pad, "Handling query {:?}", query);
 
-        stream.srcpad.peer_query(query)
+        stream.srcpad.peer_query(query.into())
```
Comment (sdroege) [inline gst-plugin-togglerecord/src/togglerecord.rs]: This seems asymmetric to me. If we get a `QueryView`, `Pad::query()` and the like should also take a `QueryView`. After all, `Pad::query()` literally just calls this function here (and some minor other things).

Comment (sdroege) [reply gst-plugin-togglerecord/src/togglerecord.rs]: But at that point, `QueryView` is used everywhere and `QueryRef` seems useless?

Comment (fengalin) [reply gst-plugin-togglerecord/src/togglerecord.rs]: It's an option, but I find it convenient to pass a concrete query directly, like in function `ToggleRecord::src_query`:
``` rust
                let mut new_query = gst::Query::new_scheduling();
                let res = stream.sinkpad.peer_query(&mut new_query);
```

Comment (fengalin) [reply gst-plugin-togglerecord/src/togglerecord.rs]: But I agree this is asymmetric...

**Comment by sdroege** [comment]
This is obsolete now after https://github.com/sdroege/gstreamer-rs/pull/82 and the new world order is implemented with https://github.com/sdroege/gst-plugin-rs/commit/b7c06e0203b60fb96cc476613040a134d0c8f71b here

**Pull request closed without merging**